cpp/plugins/cucim.kit.cuslide/src/cuslide/tiff/ifd.cpp

```diff
@@ -48,17 +52,16 @@ IFD::IFD(TIFF* tiff, uint16_t index, ifd_offset_t offset) : tiff_(tiff), ifd_ind
 {
     PROF_SCOPED_RANGE(PROF_EVENT(ifd_ifd));
     auto tif = tiff->client();
-    int ret;
 
     char* software_char_ptr = nullptr;
     char* model_char_ptr = nullptr;
-    // TODO: error handling
 
-    ret = TIFFGetField(tif, TIFFTAG_SOFTWARE, &software_char_ptr);
+    // TODO: error handling
```

Comment:
Would be good to convert this to an issue for easier tracking

Reply:
Good idea. Created https://github.com/rapidsai/cucim/issues/209